torch/library.py

```diff
@@ -20,7 +20,7 @@ class Library:
 
     Args:
         ns: library name
-        kind: "IMPL" by default
+        kind: "DEF", "IMPL" (default: "IMPL")
```

Comment:
Note that my comment from the original PR still stands: we create the C++ object beofre checking that kind is valid!

Reply:
yes updated!